Backpack/src/main/res/values-night/internal.semantic.color.xml

```diff
@@ -20,9 +20,11 @@
   <color name="__switchThumbDisabled">@color/bpkBlackTint05</color>
   <color name="__starRatingStarColor">@color/bpkBlackTint05</color>
   <color name="__horizontalNavTextDefault">@color/bpkTextTertiaryDark</color>
-  <color name="__textFieldText">@color/bpkTextPrimaryDark</color>
-  <color name="__textFieldHint">@color/bpkTextTertiaryDark</color>
-  <color name="__textFieldIcon">@color/bpkTextSecondary</color>
+  <color name="__textFieldTextDisabled">@color/bpkSkyGrayTint02</color>
+  <color name="__textFieldHint">@color/bpkSkyGrayTint04</color>
+  <color name="__textFieldIcon">@color/bpkBlackTint06</color>
```

Comment:
In the new design the icon tint could be any. We don't support this for now (and no one asked us to do this so far), but I've updated default tints using grey icon samples from the design.